source/Engine.cpp

```diff
@@ -352,6 +352,9 @@ void Engine::Step(bool isActive)
 		else if(jumpCount > 0)
 			--jumpCount;
 	}
+	// Player mission npcs can react to events.
+	for(const ShipEvent &event : events)
```

Comment:
Is this not already done?
https://github.com/endless-sky/endless-sky/blob/9c1f20ade3dd8e4e3151fe8021def5a0ed8d11ae/source/MainPanel.cpp#L100-L106

Reply:
I see... I guess at the time I was making this the FLEE events weren't arriving at NPC for another reason.